prisma/migrations/20240529092207_add_synced_channels_table/migration.sql

```diff
@@ -0,0 +1,30 @@
+-- CreateEnum
+CREATE TYPE "SyncStatus" AS ENUM ('pending', 'success', 'failed');
+
+-- CreateTable
+CREATE TABLE "SyncedChannels" (
+    "id" UUID NOT NULL DEFAULT gen_random_uuid(),
+    "copilotChannelId" VARCHAR(32),
+    "slackChannelId" VARCHAR(9),
+    "slackChannelName" VARCHAR(255),
+    "status" "SyncStatus" NOT NULL,
```

Comment:
shouldnt this be nullable? initially it is null, after running sync, it is converted to pending?

Reply:
When it is created, a sync's status is initialized as "pending" in service layer  so I made this non-nullable. @sazanrjb

Reply:
pending seems like the sync initiated but not completed yet..